docs/UserGuide.md

```diff
@@ -2,9 +2,15 @@
 layout: page title: User Guide
 ---
 
-d'Intérieur is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still
-having the benefits of a Graphical User Interface (GUI). If you can type fast, d'Intérieur can get your contact
-management tasks done faster than traditional GUI apps.
+d'Intérieur is a **desktop app for interior designers to manage their contacts and projects, optimized for use via a 
+Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). 
+If you can type fast, d'Intérieur can get your contact management tasks done faster than traditional GUI apps.
+
+If you utilize other online applications for interior designing and lack a centralized, robust, and efficient customer
+management tool, this application will be able to meet your needs.
```

Comment:
Great job on capturing the limitations of online applications which can be addressed with d'Intérieur!